### PR TITLE
Feature/home page

### PR DESCRIPTION
**Pull request by ivaschenkoo**

### CODE REVIEW

**Comment by ivaschenkoo** [comment]
[Home Page Layout](https://trello.com/c/tU30Dq4L/10-home-page-layout)

**Comment by ivaschenkoo** [comment]
[Home Page Layout](https://trello.com/c/tU30Dq4L/10-home-page-layout)

**Comment by ivaschenkoo** [comment]
[Home Page Layout](https://trello.com/c/tU30Dq4L/10-home-page-layout)

**Review by kyrylo-burmelov — CHANGES_REQUESTED** [review]
- All mocked data should be in containers. Components are using only for displaying. I would suggest to split page to several containers: Next Task, Community, Feed.

containers:
-- home
---- next-task
---- community
---- feed

components:
-- home. *use this component to render main label (Hi, Rayna) and content of another containers (from props)*
Like this:
```javascript
// components/pages/home
const Home = ({ activeUserName, nextTaskContent, ... }) => {
    return ...
}

// containers/home
const Home = () => {
    // mock data here
    
    return <HomeComponent activeUserName={...} nextTaskContent={ <NextTask /> /* NextTask is container */ } />
}
```
Then mock needed data for each of these container and return needed components. In the future we will add redux state and actions in containers.

It's needed to separate components (only displaying) from containers (logic, without styles). Also you can read [article](https://medium.com/@learnreact/container-components-c0e67432e005)

- See components/index.ts. There should be import for all basic components, all pages etc.. Then you should use created import everywhere.

File: client/src/components/pages/home/components/feed/index.tsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+					<Text tagName="h4" className={styles.title}>
+						Feed
+					</Text>
+					<span className={styles.separator}></span>
+					<Text tagName="h4" className={`${styles.feedCategory} ${styles.active}`}>
+						All
+					</Text>
+					<Text tagName="h4" className={styles.feedCategory}>
+						Questions
+					</Text>
```
[the remark this answers is not in the document]
Comment (ivaschenkoo) [reply client/src/components/pages/home/components/feed/index.tsx:36]: @romanortynskyi fix it please

File: client/src/components/pages/home/components/feed/index.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+				</div>
+				{messages.map((message) => (
+					<FeedMessage key={message.id} {...message} />
+				))}
+			</div>
+			<Button text="Load more" classList={styles.loadMoreButton} />
```
[the remark this answers is not in the document]
Comment (ivaschenkoo) [reply client/src/components/pages/home/components/feed/index.tsx:42]: @romanortynskyi fix it please

File: client/src/components/pages/home/components/community/index.tsx [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+					<div className={styles.communityColumn}>
+						<Text tagName="h5">Honor</Text>
+
+						{users.map((user) => (
+							<Text key={user.id} className={styles.columnText}>
+								{user.honor}
+							</Text>
+						))}
+					</div>
+				</div>
```
[the remark this answers is not in the document]
Comment (ivaschenkoo) [reply client/src/components/pages/home/components/community/index.tsx:61]: @romanortynskyi fix it please

**Review by kyrylo-burmelov — APPROVED** [review]

**Pull request merged**